technical/medium-editor/talk-09-20-2018/README.md

```diff
@@ -131,13 +124,31 @@
 * I've popped in once or twice and tried to get things moving and then lost momentum quickly
 
 
+## Where Are They Now?
+* Me & my career
+* Noah
+* Davi
+* j0ker
+* yabwe
+
+
 
 # Lessons
 * Things to consider when working on open-source
-* Importance of vibrant community and recruiting
+* Importance of vibrant community and sharing ownership/contribution
+* Maintaining a popular project is a full-time job
+* Work and life initiatives change, always need an exit strategy
   * Davi recruited me quickly, I waited too long to do the same
-* The downside of the glamour <-- what are we really trying to say here?  I think you want the last slide and bullet to capture the whole journey as something that was a great accelerator for the business, a rewarding personal journey, but something that left you with lingering feelings of guilt / regret
 
 
 
+# tl; dl; (too long; didn't listen;)
+* We found and were invited to take over most starred text-editor on Github
+* Lots of contributors, publicity, glamour, etc.
+* Eventually the work team moved on but I couldn't
+* I did too much on my own - had to chose between my job or the project
+* Shame, regret, soul-searching, tears, and a Developer Days Talk (so meta)
```

Comment:
I think you want a line above this one saying something like "In the end, I abandoned it, and I didn't hand it off as well as it was handed to me."